libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -241,7 +241,7 @@
 am_dsk_openmp_flush = EventFrame(
     name = "am_dsk_openmp_flush",
     entity = "on-disk OpenMP flush event",
-    comment = "Event signalizing encounter of flush construct",
+    comment = "Memory flush event triggered by OpenMP flush constructs",
```

Comment:
This change and all of the following changes in this commit are relative to code that was introduced in the previous commits. Please apply the changes to the previous commits and rebase.

Reply:
Resolved after commits got squashed.